pkg/apis/aquasecurity/v1alpha1/vulnerability_types.go

```diff
@@ -17,208 +11,6 @@ const (
 	VulnerabilityReportListKind   = "VulnerabilityReportList"
 )
 
-var (
-	// TODO Once we migrate to Go 1.16 we can use the embed package to load the CRD from ./deploy/crd/vulnerabilityreports.crd.yaml
-	VulnerabilityReportsCRD = apiextensionsv1.CustomResourceDefinition{
```

Comment:
This Go struct literal was duplicate of deploy/crd/vulnerabilityreports.crd.yaml